src/Libraries/RevitNodes/Elements/Views/ScheduleView.cs

```diff
@@ -76,7 +77,13 @@ private void InitScheduleView(Category category, string name, ScheduleType type)
             var doc = DocumentManager.Instance.CurrentDBDocument;
             TransactionManager.Instance.EnsureInTransaction(doc);
 
-            var vs = CreateViewSchedule(category, name, type);
+            // Get existing view if possible
+            var vs = ElementBinder.GetElementFromTrace<Autodesk.Revit.DB.ViewSchedule>(doc);
```

Comment:
yeah, that binds to the view so that when re-opening a graph it doesn't try to create a new one again.